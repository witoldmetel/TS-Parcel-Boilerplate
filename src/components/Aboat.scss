#about {
  padding: 40px;
  text-align: center;

  p {
    font-size: 1.2rem;
    max-width: 600px;
    margin: auto;
  }

  h2 {
    margin: 30px 0;
    color: var(--primary-color);
  }

  .social a {
    margin: 0 5px;
  }
}
