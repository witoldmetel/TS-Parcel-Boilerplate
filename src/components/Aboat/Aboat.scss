#about {
  padding: 40px;
  text-align: center;
  background-color: var(--primary-color);

  p {
    font-size: 1.2rem;
    max-width: 600px;
    margin: auto;
  }

  h2 {
    margin: 30px 0;
    color: var(--primary-color);
  }

  .about-section,
  .social-section {
    background: rgba(255, 255, 255, 0.1);
    padding: 20px;
    margin-bottom: 20px;
    border-radius: 10px;

    h1 {
      color: var(--secondary-color);
    }

    p {
      color: var(--text-color);
    }

    a {
      margin: 0 5px;
    }
  }
}
