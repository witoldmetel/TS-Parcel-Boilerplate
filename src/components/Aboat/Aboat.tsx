import React, { Component } from 'react';

import './Aboat.scss';

export class Aboat extends Component {
  render() {
    return (
      <div id="about">
        <h1>About</h1>
        <p>
          This is a minimal boilerplate example using React, TypeScript and Parcel along with Sass support. It also
          includes UIkit.
        </p>
        <h2>Follow Me On Social Media</h2>
        <div className="social">
          <a href="https://github.com/witoldmetel" uk-icon="github" target="_blank"></a>
          <a href="https://www.linkedin.com/in/witoldmetel" uk-icon="linkedin" target="_blank"></a>
        </div>
      </div>
    );
  }
}
