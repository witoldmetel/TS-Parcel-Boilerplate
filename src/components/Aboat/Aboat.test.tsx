import * as React from 'react';
import { shallow } from 'enzyme';

import { Aboat } from './Aboat';

describe('<Aboat>', () => {
  it('renders without crashing', () => {
    shallow(<Aboat />);
  });

  it('renders correctly', () => {
    const wrapper = shallow(<Aboat />);

    expect(wrapper).toMatchSnapshot();
  });
});
