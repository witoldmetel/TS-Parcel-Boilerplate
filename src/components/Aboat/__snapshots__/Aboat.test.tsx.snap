// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aboat> renders correctly 1`] = `
<div
  id="about"
>
  <h1>
    About
  </h1>
  <p>
    This is a minimal boilerplate example using React, TypeScript and Parcel along with Sass support. It also includes UIkit.
  </p>
  <h2>
    Follow Me On Social Media
  </h2>
  <div
    className="social"
  >
    <a
      href="https://github.com/witoldmetel"
      target="_blank"
      uk-icon="github"
    />
    <a
      href="https://www.linkedin.com/in/witoldmetel"
      target="_blank"
      uk-icon="linkedin"
    />
  </div>
</div>
`;
